src/Microsoft.DncEng.SecretManager/SecretTypes/GitHubAppSecret.cs

```diff
@@ -14,19 +14,19 @@ public class GitHubAppSecret : SecretType<GitHubAppSecret.Parameters>
     {
         public class Parameters
         {
-            public string AppName { get; set; }
             public bool HasPrivateKey { get; set; }
             public bool HasWebhookSecret { get; set; }
-            public bool HasAppSecret { get; set; }
+            public bool HasOAuthSecret { get; set; }
 
         }
 
         private const string AppId = "-app-id";
         private const string AppPrivateKey = "-app-private-key";
-        private const string AppSecret = "-app-secret";
-        private const string AppHook = "-app-hook";
+        private const string OAuthId = "-oauth-id";
+        private const string OAuthSecret = "-oauth-secret";
+        private const string AppHookSecret = "-app-webhook-secret";
 
-        private readonly List<string> _suffixes = new List<string> { AppId, AppPrivateKey, AppSecret, AppHook };
+        private readonly List<string> _suffixes = new List<string> { AppId, AppPrivateKey, OAuthId, OAuthSecret, AppHookSecret };
```

Comment:
It looks like in status web it's somehow mixed or named differently - I created  GitHubAppSecret and GitHubOAuthSecret to differentiate between app and oauth. But it's probably easier to have it in one secret type. I think we can now remove GitHubOAuthSecret.

Reply:
Ahh yea, I didn't see that other one. That makes sense. I don't know if we want to support old-style oauth apps, but github apps contain both oauth stuff, and github app stuff so they need both.